src/renderer/i18n/en/app.json

```diff
@@ -1183,6 +1183,11 @@
   "RecipientField": {
     "placeholder": "Enter {{currencyName}} address"
   },
+  "Terms": {
+    "title": "Terms of Use",
+    "read": "Read the Terms of Use",
+    "switchLabel": "I have read and agree with the Terms of Use and Privacy Policy"
+  },
```

Comment:
is there any reason why it was not yet in this translation file? weird.

Reply:
i'm going to merge this regardless, we can see later the diff with lldv1

Reply:
According to what i understood from @IAmMorrow the port started before the implementation of the terms in lldv1 and thus the locales we're missing.